examples/fuse_examples/imaging/classification/stoic21/README.md

```diff
@@ -0,0 +1,14 @@
+# STOIC21 Classification with Fuse 
+
+Example of severe COVID-19 classifier baseline given a Computed-Tomography (CT), age group and gender.
+More details about the challenge can be found here: https://stoic2021.grand-challenge.org/
+
+# Data
+The STOIC Dataset (see https://pubs.rsna.org/doi/10.1148/radiol.2021210384 for a full description) contains Computed Tomography scans from 10,735 patients.  For this challenge, one CT scan from each patient has been selected, and the dataset has been divided randomly into a public training set (2,000 patients), a test set (~1,000 patients)
```

Comment:
the rest is private *taining* set? can you clarify this? it means it's training set that only people with authorization can use?